src/install.c

```diff
@@ -90,12 +90,19 @@ gboolean determine_slot_states(GError **error)
 	mountlist = g_unix_mounts_get(NULL);
 	for (GList *l = mountlist; l != NULL; l = l->next) {
 		GUnixMountEntry *m = (GUnixMountEntry*)l->data;
-		g_autofree gchar *devicepath = resolve_loop_device(g_unix_mount_get_device_path(m));
-		RaucSlot *s = find_config_slot_by_device(r_context()->config,
-				devicepath);
-		if (s) {
-			s->ext_mount_point = g_strdup(g_unix_mount_get_mount_path(m));
-			g_debug("Found external mountpoint for slot %s at %s", s->name, s->ext_mount_point);
+		/* Relative paths will be from something like tmpfs, cgroups,
+		 * etc. which we want to ignore rather than try to follow some
+		 * random symlink named "tmpfs" in the CWD.  */
+		if (g_unix_mount_get_device_path(m)[0] == '/') {
```

Comment:
Following the convention in the rest of the code and to reduce nesting depth, I would prefer having a

``` c
if (g_unix_mount_get_device_path(m)[0] != '/')
        continue
```

here. Maybe then we can also remove the unreadable line break after in assignment. We do not depend that strictly on code line length.


The rest looks fine for me now! Sorry for being a bit unresponsive, but we are quite busy on all fronts 'at the moment' thus struggling sometimes to follow all updates as they would deserve :/

I've also marked this PR as 1.1 release-critical now.